src/tools/ceph_osdomap_tool.cc

```diff
@@ -31,7 +31,7 @@ int main(int argc, char **argv) {
   desc.add_options()
     ("help", "produce help message")
     ("omap-path", po::value<string>(&store_path),
-     "path to mon directory, mandatory (current/omap usually)")
+     "path to osd directory, mandatory (current/omap usually)")
```

Comment:
i'd suggest s/mon/omap/ instead.

Reply:
Really thanks your comment, and I will check it again.